content/vault/v1.16.x/content/api-docs/system/storage/raftautosnapshots.mdx

```diff
@@ -100,21 +106,25 @@ environment variables or files on disk in predefined locations.
 - `google_service_account_key` `(string)` - Google service account key in JSON format. Depending
   on how the API is invoked, this may be need to be JSON-escaped, e.g. for newlines and double quotes.
   The raw value looks like this:
-
-```json
-{
-  "type": "service_account",
-  "project_id": "project-id",
-  "private_key_id": "key-id",
-  "private_key": "-----BEGIN RSA PRIVATE KEY-----\nMIIEpQ ... /WZs=\n-----END RSA PRIVATE KEY-----\n",
-  "client_email": "service-account-email",
-  "client_id": "client-id",
-  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
-  "token_uri": "https://accounts.google.com/o/oauth2/token",
-  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
-  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/service-account-email"
-}
-```
+  ```json
+  {
+    "type": "service_account",
+    "project_id": "project-id",
+    "private_key_id": "key-id",
+    "private_key": "-----BEGIN RSA PRIVATE KEY-----\nMIIEpQ ... /WZs=\n-----END RSA PRIVATE KEY-----\n",
+    "client_email": "service-account-email",
+    "client_id": "client-id",
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://accounts.google.com/o/oauth2/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/service-account-email"
+  }
+    ```
+  
+  Make sure that the Google service account used has the following permissions in Google Cloud Storage, so that Vault can store and manage the snapshots:
```

Comment:
```suggestion
  Your Google service account must have the following permissions in Google Cloud Storage so that Vault can store and manage the snapshots:
```
Style correction: state requirements plainly